src/sdk/model/edge/certificates/__json__/crl-json.ts

```diff
@@ -0,0 +1,11 @@
+import { X509CRLJson } from './x509-crl-json';
+
+export interface CRLJson {
+  object_id: string;
+  description: string;
+  scope: string;
+  is_temporal: boolean;
+  issuer_cn: string;
+  pem_encoding: string;
+  x509_c_r_l: X509CRLJson;
```

Comment:
i think this is a typo error isn't it ? It should be `x509_crl`.

Reply:
@JoMas971 Field is returned like that from API, it has to be updated core side.

Reply:
@fradinni yeah, in those cases you need to tell the core team to let them update on their side. 
It's a JSON format issue core side ;-).

Reply:
@JoMas971  Good catch or the permission,  just added it.

Reply:
indeed, @JoMas971  is also right about the format of that JSON parameter being a bug. Please let Collin know when you see issues like that so he can fix them. I'll tell him about this one.

Reply:
@JoMas971  yeah i know. I noticed it when john pointed it out, i haven't seen this one was misformatted. Thanks @cfsnyder .